<?xml version="1.0"?>
<launch>
    <node pkg="drone_marker_pkg" name="marker_detect_node" type="video_test.py" output="screen">
    </node>

    <node pkg="image_view" type="image_view" name="image_view" output="screen">
        <remap from="image" to="/marker_detect/image_raw"/>
    </node>
</launch>
